docs/eventing/event-registry.md

```diff
@@ -14,7 +14,7 @@ type information in the cluster data store.
 
 ## Before you begin
 
-1. Read about the [broker](./broker/) and [trigger](./triggers/) objects.
+1. Read about the [broker](./broker/) and [trigger](./broker/triggers/) objects.
```

Comment:
```suggestion
1. Read about the [broker](../broker/) and [trigger](../broker/triggers/) objects.
```
I just tested locally and the change is to have it `../` 
Interesting markdown usually handles this with `./`
comments @julz ?

Reply:
this is because you're using a relative link to what the markdown file happens to end up building to rather than linking to the markdown file itself.